web/src/app/pes-admin/provider/provider.component.html

```diff
@@ -1,3 +1,85 @@
-<p>
-  Formulario de proveedor funciona!
-</p>
+<div class="cabecera">
+  <h1>Datos Proveedor</h1>
+  <button mat-raised-button (click)='onBack()' color="primary">Regresar</button>
+</div>
+<mat-spinner class="spinner" *ngIf="providerInEdition == null"></mat-spinner>
+<form class="form" *ngIf="providerInEdition != null">
+  <mat-form-field class=" full-width">
+    <input type="nombre" placeholder="Nombre" matInput [value]=providerInEdition.nombre
```

Comment:
`<input type="nombreContacto" ...`
[Aquí están los valores permitidos](https://www.w3schools.com/html/html_form_input_types.asp) para el atributo type de un input. 
Recomiendo lo siguiente para hacer en todos tus input:
- search and replace type por name en todos los inputs (es muy bueno poner el atributo name y poner como valor lo que representa)
- agregar el type="text" en los inputs que son de texto

-